privaterelay/settings.py

```diff
@@ -547,6 +549,22 @@ def _get_initial_middleware():
     'DEFAULT_RENDERER_CLASSES': DRF_RENDERERS,
 }
 
+CORS_ALLOWED_ORIGINS = []
+CSRF_TRUSTED_ORIGINS = []
+if DEBUG:
+    # In local development, the React UI can be served up from a different server
+    # that needs to be allowed to make requests.
+    # In production, the frontend is served by Django, is therefore on the same
+    # origin and thus has access to the same cookies.
+    CORS_ALLOW_CREDENTIALS = True
+    SESSION_COOKIE_SAMESITE = None
+    CORS_ALLOWED_ORIGINS += [
+        'http://localhost:3000',
+    ]
+    CSRF_TRUSTED_ORIGINS += [
+        'http://localhost:3000',
+    ]
+
```

Comment:
I think this is fine, but I might want a bit of time to think about how bad the failure mode is here. E.g., if `DEBUG` is somehow set to true in prod, what additional vulnerabilities does this open up?

Reply:
Yeah that sounds good. I think the main failure mode is DEBUG being set to true and a malicious app on the user's machine runs a server on localhost:3000. If they obtain the user's API token, they will then be able to act on behalf of that user. (Alternative attack method would be to insert itself into the browser, of course.)

Is there anyone with more domain knowledge we could ask here for a second opinion?

Reply:
At that level of ask it would probably involve a consulting fee. 😄

Reply:
This sounds like the default port for all React apps during development, and those apps are likely running without security protections? So, I'm worried about the React developer who is also a Relay user, and an attacker crafts a malicious request at their http://localhost:3000 site AND we're running in `DEBUG=True` mode.

Seems like a low-risk scenario.